docs/elements/reference/common.mdx

```diff
@@ -0,0 +1,444 @@
+
+</Callout>
+
+### Properties {{ toc: false }}
+
+| Name       | Type                                                          | Description                                                                      |
+| ---------- | ------------------------------------------------------------- | -------------------------------------------------------------------------------- |
+| `asChild` | `boolean` (optional) | If `true`, `<Icon>` will render as its child element, passing along any necessary props. Default: `false` |
+
+### Examples {{ toc: false }}
+
+#### Basic
+
+`<Icon>` will render as the Google logo here.
+
+```tsx filename="page.tsx" {4}
+<SignIn.Root>
+  <SignIn.Step name="start">
+    <Clerk.Connection name="google">
+      <Clerk.Icon />
+      Sign in with Google
+    </Clerk.Connection>
+  </SignIn.Step>
+</SignIn.Root>
+```
+
+## `<Input>`
+
+Handles rendering of `<input />` elements within Clerk's flows. Supports special `type` prop values to render input types that are unique to authentication and user management flows. Additional props will be passed through to the [`<input />` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
+
+### Properties {{ toc: false }}
+
+| Name    | Type                                                  | Description                                                                                                                                           |
+| ------- | ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `asChild` | `boolean` (optional)                                  | If true, `<Input>` will render as its child element, passing along any necessary props. Default: `false`                                                             |
+| `name`    | `string` (optional)                                   | Name for form field, will be used to associate all underlying field elements. Can be automatically inferred from the name on the `<Field>` component. Default: Automatically inferred |
+| `type`    | `text \| email \| tel \| otp* \| password` (optional) | Type for the input. Default: `text`                                                                                                                   |
+
+_Values denoted with \* are unique Clerk input types._
+
+The following data attributes are also added to the underlying Element:
+
+- `data-valid` - If the field state is valid
+- `data-invalid` - If the field state is invalid (has an associated error)
+- `date-state` - Refers to the [`<FieldState>`](#field-state) status
+
+### Examples {{ toc: false }}
+
+#### Basic
+
+Make sure to always include a [`<Label>`](#label) to correctly announce your `<Input>` to screen readers.
```

Comment:
```suggestion
```
We can delete this line, as I added it under the first paragraph of the `<Input>` section above